app/experimenter/normandy/tasks.py

```diff
@@ -163,6 +165,17 @@ def update_population_percent(experiment, recipe_data):
             experiment.save()
 
 
+def update_is_high_population(experiment, recipe_data):
+    recipe_args = recipe_data.get("arguments", {})
+    if recipe_data and ("isHighPopulation" in recipe_args):
+        recipe_is_high_population = recipe_args["isHighPopulation"]
```

Comment:
Oh you know you could do

```py
if recipe_data and recipe_is_high_population := recipe_args.get("isHighPopulation")
```

with the fancy walrus thing

![walrus-scratch](https://user-images.githubusercontent.com/119884/93358617-b601f180-f80f-11ea-9f26-6d7d550960ff.gif)